BrainPortal/db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20170308220959) do
+ActiveRecord::Schema.define(:version => 20170306210331) do
```

Comment:
This schema contains all sorts of changes completely unrelated to this pull request. Please checkout the proper schema and ONLY integrate the changes that are relevant